Planets and Systems/Asranda.md

```diff
@@ -65,7 +65,7 @@ In the wake of the disaster, life changed on the surface. The raid was blamed on
 
 It is only by coincidence that the Randians were discovered again. In 2555, a few dozen freelance miners discovered A'sranda, and landed near a particularly rich mineral deposit. Having detected no signs of life or civilization through the atmosphere, the miners were caught unprepared and were captured by a security patrol. At least one of the minres managed to get a distress call off through the heavy interference, and the Sol Central Government dispatched a task force to investigate. After making more formal contact with the surface, the situation remained tense for some time, until SolGov recruited a Resomi translator and managed to defuse the situation. The miners were released, and terse diplomatic ties were established.
 
-Today the small Republic of A’sranda remains somewhat isolationist. They have limited contact with outsiders and many of the planet's populace are distrustful of non-Resomii due to their encounter with the Vox, but relations are improving. NanoTransen, with the blessing of SolGov, has built a trading outpost within the largest surface town, and a brisk trade exists, exchanging Randian minerals for more advanced technology and mechanical expertise. Through this outpost, some Randians have trave Some Resomi from here have even managed to get offworld and see the stars on contract with NanoTrasen, or another Trans-Stellar.
+Today the small Republic of A’sranda remains somewhat isolationist. They have limited contact with outsiders and many of the planet's populace are distrustful of non-Resomii due to their encounter with the Vox, but relations are improving. NanoTrasen, with the blessing of SolGov, has built a trading outpost within the largest surface town, and a brisk trade exists, exchanging Randian minerals for more advanced technology and mechanical expertise. Some Resomi from here have even managed to get offworld and see the stars on contract with NanoTrasen, or another Trans-Stellar.
```

Comment:
"Today, the small Republic of A'sranda  sits mostly in isolation. Limited contact with outsiders, and an incredibly distrustful populace due to encounters with Vox have led to this situation, but situations are improving; slowly. NanoTrasen, with Sol Gov authorization, has been allowed to build a trading outpost in the Republic, creating a shallow yet flourishing trade. Randian minerals and advanced technology pass through the trading outpost on a weekly basis. Some Resomi have even managed to get offworld to see the stars, with a contract from a Trans-Stellar Corporation of course."